packages/headless/src/controllers/facets/range-facet/headless-range-facet.ts

```diff
@@ -44,15 +44,12 @@ export function buildRangeFacet<
   return {
     ...controller,
 
-    toggleSelect: (selection: RangeFacetValue) =>
```

Comment:
Remove that method as it was not used, and the controller itself isn't exported, so it's not a breaking change